base4kids2.ldif

```diff
@@ -498,6 +498,43 @@ attributeTypes: (
       DESC 'Determines whether a base4kids object is disabled but still active'
       SUP base4kidsIsBlocked )
 #
+# Name:     base4kidsTelephoneNumberBusiness
+# Syntax:   Phone number
+# Examples: +01234567890
+attributeTypes: (
+      1.3.6.1.4.1.31534.2.5.2.42
+      NAME 'base4kidsTelephoneNumberBusiness'
+      DESC 'Business telephone number'
+      EQUALITY telephoneNumberMatch
+      ORDERING telephoneNumberOrderingMatch
+      SYNTAX 1.3.6.1.4.1.1466.115.121.1.50
+      SINGLE-VALUE )
+#
+# Name:     base4kidsDateOfBirth
+# Syntax:   Generalized Time
+# Examples: 19951221000000Z
+attributeTypes: (
+      1.3.6.1.4.1.31534.2.5.2.43
+      NAME 'base4kidsDateOfBirth'
+      DESC 'Date of Birth'
+      EQUALITY generalizedTimeMatch
+      ORDERING generalizedTimeOrderingMatch
+      SYNTAX 1.3.6.1.4.1.1466.115.121.1.24
+      SINGLE-VALUE )
+#
+# Name:     base4kidsPlaceOfOrigin
+# Syntax:   Directory String
+# Examples: "Bern"
+attributeTypes: (
+      1.3.6.1.4.1.31534.2.5.2.44
+      NAME 'base4kidsPlaceOfOrigin'
+      DESC 'The name of a place (city)'
```

Comment:
Please change the description from `The name of a place (city)` to the `The place of origin`.